pkg/cli/config/config.go

```diff
@@ -139,7 +144,19 @@ func (config *BackplaneConfiguration) getFirstWorkingProxyURL(s []string) string
 	return ""
 }
 
-func GetConfigDirctory() (string, error) {
+func validateConfig() error {
+	globalOpts := &globalflags.GlobalOptions{}
+
+	// Validate the proxy url
+	if viper.GetStringSlice("proxy-url") == nil && os.Getenv(info.BackplaneProxyEnvName) == "" && globalOpts.ProxyURL == "" {
+		return fmt.Errorf("proxy-url must be set explicitly in either config file or via the environment HTTPS_PROXY or the global option --proxy")
```

Comment:
I have concerns on making proxy mandatory. There are cases where we don't need/want a proxy. Eg, for testing environment or Fedramp environment.

Reply:
Agree with you that it should not be mandatory, anyway it is Warning for now https://github.com/openshift/backplane-cli/pull/420/files#diff-4e5b97588faff423da8ddd605ee2a4760f4f168d9bdb8ad220d4adc5823ef086R67-R68
The plan is to make the field required, but it seems for some use cases, we will still need the proxy unset.

Reply:
ah, ok, I missed the `logger.Warn(err)`.